tests/provisioner_with_invalid_backend_sc.go

```diff
@@ -89,57 +86,27 @@ var _ = Describe("TEST NFS PROVISIONER WITH INVALID BACKEND SC", func() {
 				WithStorageClass(scName).
 				WithAccessModes(accessModes).
 				WithCapacity(capacity).Build()
-			Expect(err).To(BeNil(), "while building pvc {%s} in namespace {%s}", pvcName, applicationNamespace)
+			Expect(err).To(BeNil(), "while building pvc object for %s/%s", applicationNamespace, pvcName)
 
 			By("creating above pvc")
 			err = Client.createPVC(pvcObj)
-			Expect(err).To(BeNil(), "while creating pvc {%s} in namespace {%s}", pvcName, applicationNamespace)
-
-			pvcPhase, err := Client.waitForPVCBound(applicationNamespace, pvcName)
-			Expect(err).To(BeNil(), "while waiting for pvc %s/%s bound phase", applicationNamespace, pvcName)
-			Expect(pvcPhase).To(Equal(corev1.ClaimBound), "pvc %s/%s should be in bound phase", applicationNamespace, pvcName)
-		})
-	})
-
-	When("verifying nfs-server state", func() {
-		It("should have nfs-server in pending state", func() {
-			By("fetching nfs-server deployment name")
-			pvcObj, err := Client.getPVC(applicationNamespace, pvcName)
-			Expect(err).To(BeNil(), "while fetching pvc {%s} in namespace {%s}", pvcName, applicationNamespace)
-
-			nfsDeployment := fmt.Sprintf("nfs-%s", pvcObj.Spec.VolumeName)
-			podList, err := Client.listPods(openebsNamespace, fmt.Sprintf("%s=%s", nfsServerLabel, nfsDeployment))
-			Expect(err).To(BeNil(), "while fetching nfs-server pod")
-			Expect(podList.Items[0].Status.Phase).To(Equal(corev1.PodPending), "while verifying nfs-server pod state")
-
-			var unboundPVCCondFound bool
-			for _, v := range podList.Items[0].Status.Conditions {
-				if strings.Contains(v.Message, "pod has unbound immediate PersistentVolumeClaims") {
-					unboundPVCCondFound = true
-				}
-			}
-			Expect(unboundPVCCondFound).Should(BeTrue(), "while checking unbound PVC condition for nfs-server pod")
+			Expect(err).To(BeNil(), "while creating pvc %s/%s", applicationNamespace, pvcName)
 		})
 	})
 
-	When("verifying backend PVC state", func() {
-		It("should have backend in pending state", func() {
+	When("verifying NFS PVC state", func() {
+		It("should have NFS PVC in pending state", func() {
 			pvcObj, err := Client.getPVC(applicationNamespace, pvcName)
-			Expect(err).To(BeNil(), "while fetching pvc {%s} in namespace {%s}", pvcName, applicationNamespace)
-
-			backendPVCName := "nfs-" + pvcObj.Spec.VolumeName
-			backendPvcObj, err := Client.getPVC(openebsNamespace, backendPVCName)
-			Expect(err).To(BeNil(), "while fetching backend pvc {%s} in namespace {%s}", backendPVCName, openebsNamespace)
-			Expect(backendPvcObj.Status.Phase).To(Equal(corev1.ClaimPending), "while verifying backed PVC claim phase")
+			Expect(err).To(BeNil(), "while fetching pvc %s/%s", applicationNamespace, pvcName)
+			Expect(pvcObj.Status.Phase).To(Equal(corev1.ClaimPending), "while verifying NFS PVC claim phase")
```

Comment:
Can we add a check for event? To make sure pending is due to the code that we inserted/modified?